### PR TITLE
[Php80] Fix PHPStan on AnnotationToAttributeRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -521,17 +521,21 @@ parameters:
             path: rules/Php74/Rector/Property/TypedPropertyRector.php
 
         -
-            message: '#Class cognitive complexity is 55, keep it under 30#'
+            message: '#Class cognitive complexity is 31, keep it under 30#'
```
Comment (samsonasik) [inline phpstan.neon]: @TomasVotruba this is the improvement I can made, I think the other logics complexity are essential to be kept as is.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: phpstan.neon (hunk 1 above)
Comment (samsonasik) [reply phpstan.neon]: resolved by moved resolve DoctrineAnnotationTagValueNode to `DoctrineAnnotationTagValueNodeAnalyzer` service

**Comment by samsonasik** [comment]
@TomasVotruba finally fixed 🎉🎉🎉

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**